packages/shared/lib/core/profile/actions/profiles/checkAndMigrateProfiles.ts

```diff
@@ -331,3 +333,23 @@ function persistedProfileMigrationToV17(existingProfile: IPersistedProfile): voi
     existingProfile.network.chains = newChains
     saveProfile(existingProfile)
 }
+
+function persistedProfileMigrationToV18(existingProfile: IPersistedProfile): void {
+    if (existingProfile.network) {
+        interface IOldPersistedNetwork {
+            protocol: unknown
+            baseToken: unknown
+        }
+
+        const oldNetwork = existingProfile.network as unknown as IOldPersistedNetwork
+        delete oldNetwork.protocol
+        delete oldNetwork.baseToken
+
+        const newNetwork = oldNetwork as unknown as IPersistedNetwork
+        const defaultBech32Hrp = DEFAULT_NETWORK_METADATA[existingProfile.network.id]?.bech32Hrp || ''
```

Comment:
In what cases would `DEFAULT_NETWORK_METADATA[existingProfile.network.id]?.bech32Hrp` not exist? I just because it seems weird to me to leave `''` as fallback value, it will likely break if that ever happens.